bk-config/create_client_files.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
I think you should run this script within the `bk-config` container like ure other scripts are run.

To do this you can make the following changes

1. run the script with a volume mount of the current working directory to `/output` and then in your shell script copy the created files to /output to have them show-up on the the user's native system: `docker run --rm --network beekeeper_default --name bk-config -v beekeeper-config_bk-secrets:/usr/lib/sage/:ro -v 'pwd':/output sagecontinuum/bk-config create_client_files.sh param1 param2 param3
2. update this script to execute the `create_known_hosts_file.sh` and `create_registeration_cert.sh` scripts and capture their output and the copied secret files to /output

Reply:
yes, in hindsight, that seems better.